### ISSUE TITLE
Delta invest Tracker Converter

### ISSUE THREAD

**Issue by michi1996** [dickwolff/Export-To-Ghostfolio#126]
I would like to import my Delta Investment Tracker export into Ghostfolio. Although the app is from etoro, the converter unfortunately doesn't work. Can this be added?

**Comment by dickwolff** [comment]
Of course, I can look into this. Can you provide me an example export of Delta?

**Comment by michi1996** [comment]
Hi, thanks for the fast answer. You can find a Template Export at the website from Delta.
https://support.delta.app/en/articles/8001646-import-from-the-delta-csv-template

Let me know when you need additional infos.

**Comment by dickwolff** [comment]
@michi1996 I created an intial version with only Stock support. Could you test this for me? You can pull [this](https://hub.docker.com/layers/dickwolff/export-to-ghostfolio/0.20.0-beta/images/sha256-ff780d89f5023728adfe46ee607159cce578db5f7aca543c687c0b3a1af333c9?context=explore) image.

I left Crypto out for now, because (for one) searching the symbol will be a challenge, but also fees could be in any other currency and for Ghostfolio to work the right way you need the same currency as your order (so a BUY BTC should have BTC fees and not BNB).

**Comment by dickwolff** [comment]
Merged in release 0.20.0
